docs.html

```diff
@@ -620,7 +647,35 @@ <h2 class="component-heading">Slider</h2>
                 <pre></pre>
                 <hr>
             </section>
+            <!-- Modal Components -->
+            <section id="modal">
+                <h2 class="component-heading">Modal</h2>
+                <p class="component-desc">Open/Close modal </p>
+                <!-- Trigger/Open The Modal -->
```

Comment:
make text color white, it will match to **modal header** and **modal footer** color

Reply:
Okay bro thanks for suggestion🚀